tests/playbooks/roles/install-devstack/templates/local.conf.j2

```diff
@@ -0,0 +1,113 @@
+[[local|localrc]]
+RECLONE=False
+HOST_IP={{ local_ip_address }}
+DEST=/opt/stack
+DATA_DIR=${DEST}/data
+USE_PYTHON3=True
+LOGFILE=$DEST/logs/stack.sh.log
+VERBOSE=True
+LOG_COLOR=False
+LOGDAYS=1
+SERVICE_TIMEOUT=300
+
+DATABASE_PASSWORD=password
+ADMIN_PASSWORD=password
+SERVICE_PASSWORD=password
+SERVICE_TOKEN=password
+RABBIT_PASSWORD=password
+
+TARGET_BRANCH={{ branch }}
+
+ENABLED_SERVICES=rabbit,mysql,key
+
+{% if "nova" in enable_services %}
+# Nova
```

Comment:
Do we need to enable the below nova services explicitly. Aren't they by default enabled?

Reply:
IIUC, when we use `ENABLED_SERVICES=xxx`, all the services should be enabled explicitly.

Reply:
ok